src/js/media-loading-indicator.ts

```diff
@@ -207,6 +208,30 @@ class MediaLoadingIndicator extends globalThis.HTMLElement {
   set mediaLoading(value: boolean) {
     setBooleanAttr(this, MediaUIAttributes.MEDIA_LOADING, value);
   }
+
+  get mediaController(): string | undefined {
+    return this.getAttribute(MediaStateReceiverAttributes.MEDIA_CONTROLLER);
+  }
+
+  set mediaController(value: string | undefined) {
+    if (value) {
+      this.setAttribute(MediaStateReceiverAttributes.MEDIA_CONTROLLER, '');
```

Comment:
must be a string value here. might as well use the element attribute utils here for all methods too